meta-lmp-base/classes/lmp-disable-gplv3.bbclass

```diff
@@ -7,4 +7,7 @@ PACKAGECONFIG:remove:pn-bluez5 = "readline mesh"
 PACKAGECONFIG:append:pn-bluez5 = " libedit "
 PACKAGECONFIG:remove:pn-iproute2 = "elf"
 PACKAGECONFIG:remove:pn-wireguard-tools = "bash-completion wg-quick"
+PACKAGECONFIG:remove:pn-networkmanager = "readline ifupdown"
+PACKAGECONFIG:remove:pn-systemd = "vconsole"
```

Comment:
Same question, missing from commit message.

Reply:
The missing information from commit log is basically:
readline is a dependency licensed in one of the avoided license.
the other packages include a bash-completion as DEPENDS and I decided removing the packages would be easier. But, as ways, this is only a suggested way to work with this kind of dependency. We can review it in the future